app/src/main/java/ch/eureka/eurekapp/ui/meeting/MeetingDetailScreen.kt

```
@@ -0,0 +1,592 @@
+
+              if (attachments.isEmpty()) {
+                Text(
+                    text = "No attachments",
+                    style = MaterialTheme.typography.bodyMedium,
+                    color = Color.Gray,
+                    modifier = Modifier.testTag(MeetingDetailScreenTestTags.NO_ATTACHMENTS_MESSAGE))
+              } else {
+                attachments.forEach { attachment -> AttachmentItem(attachmentUrl = attachment) }
+              }
+            }
+      }
+}
+
+/** Individual attachment item. */
+@Composable
+private fun AttachmentItem(attachmentUrl: String) {
+  Row(
+      modifier = Modifier.fillMaxWidth().testTag(MeetingDetailScreenTestTags.ATTACHMENT_ITEM),
+      verticalAlignment = Alignment.CenterVertically) {
+        Icon(
+            imageVector = Icons.Default.Description,
+            contentDescription = "Attachment",
+            tint = MaterialTheme.colorScheme.primary,
+            modifier = Modifier.size(24.dp))
+        Spacer(modifier = Modifier.width(12.dp))
+        Text(text = attachmentUrl, style = MaterialTheme.typography.bodyMedium, maxLines = 1)
+      }
+}
+
+/** Action buttons section (join, record, transcript, delete). */
+@Composable
+private fun ActionButtonsSection(
+    meeting: Meeting,
+    onJoinMeeting: (String) -> Unit,
+    onRecordMeeting: () -> Unit,
+    onViewTranscript: () -> Unit,
+    onDeleteMeeting: () -> Unit,
+) {
+  Column(
+      modifier =
+          Modifier.fillMaxWidth().testTag(MeetingDetailScreenTestTags.ACTION_BUTTONS_SECTION),
+      verticalArrangement = Arrangement.spacedBy(12.dp)) {
+        Text(
+            text = "Actions",
+            style = MaterialTheme.typography.titleMedium,
+            fontWeight = FontWeight.SemiBold)
+
+        // Status-specific action buttons
+        when (meeting.status) {
+          MeetingStatus.SCHEDULED,
+          MeetingStatus.IN_PROGRESS -> {
+            if (meeting.format == MeetingFormat.VIRTUAL && meeting.link != null) {
```

Important : as before, here if we have a `VIRTUAL` meeting but here is not link we should either throw an exception, or inform the user, or best check this when loading the meeting.